cypress/support/utils.js

```diff
@@ -62,66 +58,60 @@ export const fillInProbeForm = (
 export const buttonClick = (buttonName) => {
   cy.get('button[kind="secondary"]')
     .contains(buttonName)
-    .click({force: true});
+    .click({ force: true });
 };
 
 export const buttonOnTabClick = (buttonName) => {
   cy.get('button[kind="secondaryFormSubmit"]')
     .contains(buttonName)
-    .click({force: true});
+    .click({ force: true });
 };
 
 export const buttonCheckExists = (buttonName) => {
-  cy.get('button[kind="secondary"]')
-    .contains(buttonName)
-    .should("exist");
+  cy.get('button[kind="secondary"]').contains(buttonName).should("exist");
 };
 
-function generateUUID(){
+function generateUUID() {
   const uuid = uuidv4();
   return uuid;
 }
 
-export function generateUniqueName(prefix){
+export function generateUniqueName(prefix) {
   const uuid = generateUUID();
   const uniqueName = `${prefix}_${uuid}`;
   return uniqueName;
 }
 
 export const probeDelete = (probeName) => {
-  cy.log("Deleting probe: ",  probeName);
+  cy.log("Deleting probe: ", probeName);
 
   cy.get('div[data-testid="stMarkdownContainer"]')
     .should("exist")
     .contains(probeName)
     .should("exist")
     .parents('div[data-testid="stHorizontalBlock"]') // finds all the parents of the element with probeName
     .should("exist")
-    .within(() => {  // Only searches within specific stHorizontalBlock that has probeName
+    .within(() => {
+      // Only searches within specific stHorizontalBlock that has probeName
       cy.get('div[data-testid="column"]')
         //.contains("&#x1F5D1;") // Does not work. This is unicode HTML representation of the wastebasket icon.
         .eq(-2) // Brute force solution: chose second before last column (wastebasket)
         .should("exist")
-        .click()
-    })
+        .click();
+    });
 };
 
 // If "None" - don't modify the field
-export const fillInNewUngroupedLabelForm = (
-  labelName,
-  condition
-) => {
-
-  if ( labelName != "None" ) {
-    cy.get('input[aria-label="Label Name"]')
-      .clear()
-      .type(labelName);
+export const fillInNewUngroupedLabelForm = (labelName, condition) => {
+  if (labelName != "None") {
+    cy.get('input[aria-label="Label Name"]').clear().type(labelName);
   }
 
-  if ( condition != "None" ) {
+  if (condition != "None") {
     cy.get('textarea[aria-label="Condition"]')
       .clear()
-      .type(condition).type('{command+enter}');
```

Comment:
@susannahmarcus, this one should be replaced back to be `type('{command+enter}')`.
Broke tests for me in the past.

Reply:
To the best of my knowledge, this should be fine in that formatting since it just spaced it over two lines, but from a JS perspective, this is the same thing

Reply:
that said, I need to update something that I'm seeing is failing on deploy and see if that helps...

Reply:
this is my bad, I thought it replaced **command+enter** with **command**
we are good! Sorry about that.